protobuf/acm.proto

```diff
@@ -20,15 +20,25 @@ message Account {
     option (gogoproto.goproto_stringer) = false;
     bytes Address = 1 [(gogoproto.customtype) = "github.com/hyperledger/burrow/crypto.Address", (gogoproto.nullable) = false];
     crypto.PublicKey PublicKey = 2 [(gogoproto.nullable) = false];
+    // Sequence counts the number of transactions that have been accepted from this account
     uint64 Sequence = 3;
+    // The account's current native token balance
     uint64 Balance = 4;
+    // We expect exactly one of EVMCode, WASMCode, and NativeName to be non-empty
+    // EVM bytecode
     bytes EVMCode = 5 [(gogoproto.customtype) = "Bytecode", (gogoproto.nullable) = false];
     permission.AccountPermissions Permissions = 6 [(gogoproto.nullable) = false];
+    // WASM bytecode
     bytes WASMCode = 7 [(gogoproto.customtype) = "Bytecode", (gogoproto.jsontag) = ",omitempty", (gogoproto.nullable) = false];
+    // Fully qualified name of native contract for which this Account is a sentinel
```

Comment:
I'm not sure I entirely understand this comment...

Reply:
Fair. I'll expand in code, but what I meant:

- Fully qualified as in `Permissions.setBase` - I'm working on the assumption that every function has a globally unique name defined by the contract/function or just function if it is a 'loose' function
- Sentinel in the sense that this is an account object that represents a 'special' value, a native function, and doesn't contain enough information to fully process a call to the account (unlike EVM or WASM accounts. It's a marker account (at least for now). In the sense of : https://en.wikipedia.org/wiki/Sentinel_value